### PR TITLE
deets and cfp to single column layout, vert spacing to match style guide

### PR DESCRIPTION
**Pull request by theutahkate**
Kerri and I were thinking the alignment's wonky with the two column layout, so let's go back to single column. Also changed vertical spacing to match the style guide and the rest of the page.

### CODE REVIEW

**Comment by kerrizor** [comment]
:shipit:

**Comment by khjrtbrg** [comment]
Looks good! :ship:

File: app/public/css/bootstrap-theme.css [hunk 1]
```diff
@@ -1242,7 +1242,7 @@ p {
   margin: 0 0 11px; }
 
 .lead {
-  margin-bottom: 22px;
+  margin-bottom: 0;
```
Comment (strand) [inline app/public/css/bootstrap-theme.css]: Uh, I'm really hesitant to directly modify generated styles.

File: app/public/css/override.css [hunk 2]
```diff
@@ -5,5 +5,17 @@
 .cfp .btn {
   border-radius: 0px;
   font-weight: 900;
-  font-size: 25px;
+  font-size: 25px; 
+  margin-top: 34px;
```
Comment (strand) [inline app/public/css/override.css]: Why do we need to do significant whitespace futzing?

File: app/views/index.erb [hunk 3]
```diff
@@ -18,24 +18,33 @@
...
 <div class="row">
   <div class="col-md-12">
-    <img class="confetti" src="img/lightning_confetti.png">
+    <img class="confetti" src="img/heart_confetti.png">
   </div>
 </div>
-
 <div class="row">
-  <div id="deets" class="col-md-6">
+  <div id="deets" class="col-lg-12">
```
Comment (strand) [inline app/views/index.erb]: It seems inconsistent to me to set the breakpoint to `lg`, where elsewhere we break at `md`. It's moot with `12` classes, but I think we can be more consistent with our [grid options](http://getbootstrap.com/css/#grid-options)

File: app/views/index.erb [hunk 4]
```diff
@@ -18,24 +18,33 @@
...
     <h2 class="blue">Where and When</h2>
-    <p class="purple lead"><strong>October 2nd and 3rd</strong></p>
-    <p class="purple lead"><a class="pink" href="http://wp.seattlecentral.edu/broadway-performance-hall/">Broadway Performance Hall</a><br />
+    <p class="purple lead text-center"><strong>October 2nd and 3rd</strong></p>
```
Comment (strand) [inline app/views/index.erb]: I think that centered text is inconsistent with the right-aligned text above.

File: app/public/css/bootstrap-theme.css (hunk 1 above)
Comment (khjrtbrg) [inline app/public/css/bootstrap-theme.css]: :green_heart: 09de893 Moved this and copied a previous change to `override.css`, so we can add it to the scss files later.

File: app/views/index.erb (hunk 3 above)
Comment (khjrtbrg) [inline app/views/index.erb]: :green_heart: 8bae1d2 slightly more consistent!

File: app/views/index.erb (hunk 4 above)
Comment (khjrtbrg) [inline app/views/index.erb]: Personally, I like how it looks centered here. Do you think we should center the text above, too?

File: app/public/css/override.css (hunk 2 above)
Comment (khjrtbrg) [inline app/public/css/override.css]: eba585f This is a first step towards standardizing spacing. I'll get these into the scss files in the near future!

**Comment by kerrizor** [comment]
:+1: :shipit: :arrow_forward:

**Pull request merged**